zzz_generated/app-common/app-common/zap-generated/cluster-objects.h

```diff
@@ -6455,9 +6455,9 @@ enum class Fields
 struct Type
 {
 public:
-    chip::FabricIndex fabricIndex;
-    Privilege privilege;
-    AuthMode authMode;
+    chip::FabricIndex fabricIndex = static_cast<chip::FabricIndex>(0);
+    Privilege privilege           = static_cast<Privilege>(0);
+    AuthMode authMode             = static_cast<AuthMode>(0);
```

Comment:
Auth mode 0 is invalid according to the enum (and privilege above seems like a bitset ... should it be used as a bitset?)

It seems better to have  a AuthoMode::kInvalid = 0 value and use this here instead of a static cast to a value that is invalid for the underlying type.

Reply:
I had a chat with @bzbarsky-apple before about this. We're not guaranteed to always have an 'invalid' value in an enum, or even know what a sane default for an enum should be. That is usually fairly enum-specific, and so far, not specified in the XML.

For now, this I think suffices.

Reply:
> It seems better to have a AuthoMode::kInvalid = 0 value

Where would this value come from?  See also discussion at https://github.com/project-chip/connectedhomeip/pull/13432#discussion_r782509765

Reply:
In other protocols I have seen enforcements of "first value is always the default" but that is also poorly defined in case first value changes. Ideally we would set it to 0 and fail to compile if 0 is not a valid value ... basically then forcing codegen to either have a 0 or if 0 is not available make a `_kInvalid` that is 0.

Won't block the PR on this though. When reviewing I failed to see the filename and thought this was non-generated code.